docs/branding/logos.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Add a reference to the color palette and illustrative PNG to this file. You have an appropriate palette section at the bottom where this should be placed.

Reply:
Okay got it, will resolve this immediately

Reply:
Rename this file branding.md. The logos should go on a separate page.